{"name": "输入正确username、错误password，验证用户登录失败", "status": "passed", "steps": [{"name": "登陆case", "status": "passed", "parameters": [{"name": "host", "value": "'http://47.96.25.229:7300/mock/60e29c972b9d72001731a419/test'"}, {"name": "username", "value": "'admin'"}, {"name": "password", "value": "'111111'"}], "start": 1626082567512, "stop": 1626082567542}], "attachments": [{"name": "log", "source": "888bfd41-2d1d-4d9d-a01c-6a7e819bc825-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_input", "value": "{'username': 'admin', 'password': '111111'}"}, {"name": "data_expect", "value": "{'code': 1, 'data': 'login failed'}"}, {"name": "title", "value": "'输入正确username、错误password，验证用户登录失败'"}], "start": 1626082567512, "stop": 1626082567542, "uuid": "f4f39192-67ec-4896-80d5-79953ef21457", "historyId": "daeab8a7dbe721dfe8588c9e333e4dd1", "testCaseId": "4cb0134c279d13d487923af9db52d091", "fullName": "case.test_login.TestLogin#test_login", "labels": [{"name": "feature", "value": "TestLogin"}, {"name": "story", "value": "登录接口测试用例"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "asce"}, {"name": "thread", "value": "28820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}